CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 Change Log
 ==========
+## 2.2.4 (TBD)
+
+### Added:
+- (Kotlin) Add a new parameter `generateApi` you can enable/disable to generate API with infrastructure. 
+  If you set it to false, only model classes will be generated.
```

Comment:
If you set it to false api classes will not be generated. Model to nijak neovlivňuje. Pokud bys pak měl i proměnnou na model tak tohle už nebude pravda protože generateApi != generateModel.